Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15 as builder
+
+COPY go.mod go.sum /
+RUN go mod download
+
+ADD . /code
+WORKDIR /code
+RUN make build
+
+FROM scratch
```

Comment:
Lets do alpine. Nothing worse than trying to jump into a container to debug and run tcpdump or w/e and its based off of scratch!

Reply:
hmm...a security first argument is that troubleshooting should be done by adding another container in the same namespace with tools like tcpdump/etc to do this type of troubleshooting.

Reply:
The thing is we don't have an easy way to attach containers to namespaces in the non-k8s sites. I think once we've transitioned to all k8s it makes sense to go from `scratch`

Reply:
for non-k8s platforms, the pattern is to create the initial deployment manifest with a troubleshooting container.

Reply:
sgtm